code/game/verbs/webmap.dm

```diff
@@ -0,0 +1,12 @@
+/client/verb/webmap()
```

Comment:
Should this go into `interface\interface.dm` instead of its own file? The verbs for the other buttons all seem to be in there.

Reply:
Probably